Tribler/community/multichain/community.py

```diff
@@ -282,6 +289,76 @@ def get_statistics(self, public_key=None):
             statistics["total_down"] = 0
         return statistics
 
+    @blocking_call_on_reactor_thread
+    def get_graph(self, public_key=None, neighbor_radius=1):
+        """
+        Return a dictionary with the neighboring nodes and edges of a certain focus node within a certain radius,
+            regarding the local multichain database.
+
+        :param public_key: the public key of the focus node
+        :param neighbor_radius: the radius within which the neighbors have to be returned
+        :return: a tuple of a dictionary with nodes and a dictionary with edges
+        """
+        if public_key is None:
+            public_key = self.my_member.public_key
+        list_of_nodes = self.database.neighbor_list(public_key)
+        if public_key not in list_of_nodes:
+            list_of_nodes[public_key] = {"up": self.database.total_up(public_key),
+                                         "down": self.database.total_down(public_key)}
+        nodes = []
+        for current_key in list_of_nodes:
+            nodes.append({"public_key": current_key, "total_up": self.database.total_up(current_key),
+                          "total_down": self.database.total_down(current_key)})
+            self.page_rank.add_node(current_key)
```

Comment:
Please explain in the method documentation what you are doing with the `page_rank` in this method.